packages/hash/api/src/db/adapter.ts

```diff
@@ -562,58 +576,70 @@ export interface DbClient {
   /**
    * Update the operation of an existing aggregation.
    *
-   * @param params.aggregationId the id of the aggregation
-   * @param params.operation the updated aggregation operation
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.aggregationId - the id of the aggregation
+   * @param {object} params.operation - the updated aggregation operation
+   * @param {string} params.updatedByAccountId - the account id of the user that is updating the aggregation
+   * @returns {DbAggregation} the updated aggregation
    */
   updateAggregationOperation(params: {
+    sourceAccountId: string;
     aggregationId: string;
-    operation: object;
+    updatedOperation: object;
+    updatedByAccountId: string;
   }): Promise<DbAggregation>;
 
   /**
-   * Get an aggregation by its id.
+   * Get an aggregation by its aggregation id.
    *
-   * @param params.aggregationId the id of the aggregation
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.aggregationId - the id of the aggregation
+   * @returns {DbAggregation | null} the aggregation with if found in the datastore, otherwise `null`
    */
   getAggregation(params: {
+    sourceAccountId: string;
     aggregationId: string;
   }): Promise<DbAggregation | null>;
 
   /**
    * Get an aggregation by its source entity and path.
    *
-   * @param params.sourceAccountId the account id of the source entity
-   * @param params.sourceEntityId the entity id of the source entity
-   * @param params.sourceEntityVersionId the entityVersionId of the source entity (optional)
-   * @param params.path the aggregation path
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.sourceEntityId - the entity id of the source entity
+   * @param {string} params.path - the aggregation path
+   * @param {Date} [params.activeAt] - the timestamp at which the aggregation was active, when the source entity is versioned
+   * @returns {DbAggregation | null} the aggregation if found in the datastore, otherwise `null`
```

Comment:
```suggestion
   * @returns {Promise<DbAggregation | null>} the aggregation if found in the datastore, otherwise `null`
```

Reply:
Addressed in https://github.com/hashintel/hash/commit/7ef96fd5aa9766a6db237409f1b2b58b44010647, as well as for the other methods where a promise is returned.